Rename UserAgentPartIds to UserAgentParts

annevk opened the pull request:
#### c15868c771c4a4622949700d53c70abce580564c
<pre>
Rename UserAgentPartIds to UserAgentParts
<a href="https://bugs.webkit.org/show_bug.cgi?id=267177">https://bugs.webkit.org/show_bug.cgi?id=267177</a>

Reviewed by Tim Nguyen.

This also renames:

- pseudoAttr to useragentpartAttr
- Element::pseudo() to Element::userAgentPart()
- Element::setPseudo() to Element::setUserAgentPart()
- Internals::userAgentPartId() to Internals::userAgentPart()
- Internals::setUserAgentPartId() to Internals::setUserAgentPart()

And adjusts various variables and static functions accordingly.

* LayoutTests/editing/style/apply-style-atomic-expected.txt:
* LayoutTests/editing/style/apply-style-atomic-live-range-expected.txt:
* LayoutTests/fast/dom/HTMLMeterElement/meter-clone-expected.txt:
* LayoutTests/fast/dom/HTMLMeterElement/meter-clone.html:
* LayoutTests/fast/dom/HTMLMeterElement/meter-element-markup-expected.txt:
* LayoutTests/fast/dom/HTMLProgressElement/progress-clone-expected.txt:
* LayoutTests/fast/dom/HTMLProgressElement/progress-clone.html:
* LayoutTests/fast/dom/HTMLProgressElement/progress-element-markup-expected.txt:
* LayoutTests/fast/dom/nodesFromRect/resources/nodesFromRect.js:
(checkShadowContent):
* LayoutTests/fast/forms/caps-lock-indicator-width.html:
* LayoutTests/fast/forms/resources/common.js:
(getElementByPseudoId):
* LayoutTests/fast/forms/search-field-buttons-do-not-have-focus-rings-expected.html:
* LayoutTests/fast/forms/search-field-buttons-do-not-have-focus-rings.html:
* LayoutTests/fast/forms/validation-message-clone-expected.txt:
* LayoutTests/fast/forms/validation-message-clone.html:
* LayoutTests/fast/html/clone-range-expected.txt:
* LayoutTests/fast/html/clone-range.html:
* LayoutTests/fast/html/details-clone.html:
* LayoutTests/http/tests/media/hls/hls-webvtt-style-expected.txt:
* LayoutTests/http/tests/media/hls/hls-webvtt-style.html:
* LayoutTests/http/tests/media/track/track-webvtt-vertical-multi-line-expected.txt:
* LayoutTests/http/tests/media/track/track-webvtt-vertical-multi-line.html:
* LayoutTests/inspector/css/pseudo-element-matches.html:
* LayoutTests/media/modern-media-controls/media-controller/media-controller-single-container-expected.txt:
* LayoutTests/media/modern-media-controls/media-controller/media-controller-single-container.html:
* LayoutTests/media/modern-media-controls/media-controls/media-controls-display-above-captions-expected.txt:
* LayoutTests/media/modern-media-controls/media-controls/media-controls-display-above-captions.html:
* LayoutTests/media/modern-media-controls/text-tracks/text-tracks-height-no-controls.html:
* LayoutTests/media/modern-media-controls/text-tracks/text-tracks-height-with-controls.html:
* LayoutTests/media/track/track-cue-vertical-style.html:
* LayoutTests/media/track/track-in-band-layout.html:
* LayoutTests/media/track/track-user-stylesheet-override.html:
* LayoutTests/media/track/webvtt-region-position-expected.txt:
* LayoutTests/media/track/webvtt-region-position.html:
* LayoutTests/platform/ios/ios/plugin/youtube-flash-plugin-iframe-expected.txt:
* LayoutTests/platform/ios/ios/plugin/youtube-flash-plugin-iframe.html:
* LayoutTests/plugins/quicktime-plugin-replacement-expected.txt:
* LayoutTests/plugins/quicktime-plugin-replacement.html:
* LayoutTests/resources/dump-as-markup.js:
(Markup._get):
* LayoutTests/userscripts/insert-stylesheets-with-pseudo.html:
* Source/WebCore/CMakeLists.txt:
* Source/WebCore/DerivedSources-output.xcfilelist:
* Source/WebCore/DerivedSources.make:
* Source/WebCore/Headers.cmake:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/css/CSSPseudoSelectors.json:
* Source/WebCore/css/SelectorChecker.cpp:
(WebCore::SelectorChecker::matchRecursively const):
* Source/WebCore/css/process-css-pseudo-selectors.py:
* Source/WebCore/dom/Element.cpp:
(WebCore::Element::attributeChanged):
(WebCore::Element::userAgentPart const):
(WebCore::Element::setUserAgentPart):
(WebCore::Element::pseudo const): Deleted.
(WebCore::Element::setPseudo): Deleted.
* Source/WebCore/dom/Element.h:
* Source/WebCore/dom/mac/ImageControlsMac.cpp:
(WebCore::ImageControlsMac::createImageControls):
* Source/WebCore/html/BaseDateAndTimeInputType.cpp:
(WebCore::BaseDateAndTimeInputType::createShadowSubtree):
* Source/WebCore/html/ColorInputType.cpp:
(WebCore::ColorInputType::createShadowSubtree):
* Source/WebCore/html/FileInputType.cpp:
(WebCore::FileInputType::createShadowSubtree):
* Source/WebCore/html/HTMLAttributeNames.in:
* Source/WebCore/html/HTMLMeterElement.cpp:
(WebCore::setValueClass):
(WebCore::HTMLMeterElement::didAddUserAgentShadowRoot):
* Source/WebCore/html/RangeInputType.cpp:
(WebCore::RangeInputType::createShadowSubtree):
* Source/WebCore/html/SearchInputType.cpp:
(WebCore::updateResultButtonPseudoType):
* Source/WebCore/html/TextFieldInputType.cpp:
(WebCore::TextFieldInputType::createShadowSubtree):
(WebCore::TextFieldInputType::createDataListDropdownIndicator):
(WebCore::autoFillButtonTypeToAutoFillButtonPseudoClassName):
(WebCore::isAutoFillButtonTypeChanged):
(WebCore::TextFieldInputType::createContainer):
(WebCore::TextFieldInputType::createAutoFillButton):
(WebCore::TextFieldInputType::updateAutoFillButton):
* Source/WebCore/html/ValidationMessage.cpp:
(WebCore::ValidationMessage::buildBubbleTree):
* Source/WebCore/html/shadow/DateTimeEditElement.cpp:
(WebCore::DateTimeEditBuilder::visitLiteral):
(WebCore::DateTimeEditElement::create):
(WebCore::DateTimeEditElement::layout):
* Source/WebCore/html/shadow/DateTimeFieldElements.cpp:
(WebCore::DateTimeDayFieldElement::create):
(WebCore::DateTimeHourFieldElement::create):
(WebCore::DateTimeMeridiemFieldElement::create):
(WebCore::DateTimeMillisecondFieldElement::create):
(WebCore::DateTimeMinuteFieldElement::create):
(WebCore::DateTimeMonthFieldElement::create):
(WebCore::DateTimeSecondFieldElement::create):
(WebCore::DateTimeSymbolicMonthFieldElement::create):
(WebCore::DateTimeYearFieldElement::create):
* Source/WebCore/html/shadow/DetailsMarkerControl.cpp:
(WebCore::DetailsMarkerControl::create):
* Source/WebCore/html/shadow/MediaControlTextTrackContainerElement.cpp:
(WebCore::MediaControlTextTrackContainerElement::create):
* Source/WebCore/html/shadow/ProgressShadowElement.cpp:
(WebCore::ProgressInnerElement::create):
(WebCore::ProgressBarElement::create):
(WebCore::ProgressValueElement::create):
* Source/WebCore/html/shadow/SliderThumbElement.cpp:
(WebCore::SliderThumbElement::create):
(WebCore::SliderContainerElement::create):
* Source/WebCore/html/shadow/SpinButtonElement.cpp:
(WebCore::SpinButtonElement::create):
* Source/WebCore/html/shadow/SwitchThumbElement.cpp:
(WebCore::SwitchThumbElement::create):
* Source/WebCore/html/shadow/SwitchTrackElement.cpp:
(WebCore::SwitchTrackElement::create):
* Source/WebCore/html/shadow/TextControlInnerElements.cpp:
(WebCore::TextControlPlaceholderElement::create):
(WebCore::SearchFieldCancelButtonElement::create):
* Source/WebCore/html/track/TextTrackCue.cpp:
(WebCore::TextTrackCueBox::initialize):
(WebCore::tagPseudoObjects):
(WebCore::removeUserAgentPartAttributes):
(WebCore::TextTrackCue::getCueAsHTML):
(WebCore::TextTrackCue::rebuildDisplayTree):
(WebCore::removePseudoAttributes): Deleted.
* Source/WebCore/html/track/VTTCue.cpp:
(WebCore::VTTCue::obtainCSSBoxes):
* Source/WebCore/html/track/VTTRegion.cpp:
(WebCore::VTTRegion::getDisplayTree):
(WebCore::VTTRegion::prepareRegionDisplayTree):
* Source/WebCore/inspector/agents/InspectorDOMAgent.cpp:
(WebCore::InspectorDOMAgent::highlightSelector):
* Source/WebCore/page/CaptionUserPreferencesMediaAF.cpp:
(WebCore::CaptionUserPreferencesMediaAF::captionsStyleSheetOverride const):
* Source/WebCore/rendering/RenderTheme.cpp:
(WebCore::RenderTheme::autoAppearanceForElement const):
* Source/WebCore/style/ElementRuleCollector.cpp:
(WebCore::Style::ElementRuleCollector::matchPartPseudoElementRules):
(WebCore::Style::ElementRuleCollector::collectMatchingUserAgentPartRules):
* Source/WebCore/style/RuleData.cpp:
(WebCore::Style::determinePropertyAllowlist):
* Source/WebCore/style/StyleAdjuster.cpp:
(WebCore::Style::Adjuster::adjust const):
* Source/WebCore/style/StyleInvalidator.cpp:
(WebCore::Style::Invalidator::invalidateUserAgentParts):
* Source/WebCore/style/StyleResolver.cpp:
(WebCore::Style::elementTypeHasAppearanceFromUAStyle):
* Source/WebCore/style/StyleSharingResolver.cpp:
(WebCore::Style::SharingResolver::canShareStyleWithElement const):
* Source/WebCore/testing/Internals.cpp:
(WebCore::Internals::userAgentPart):
(WebCore::Internals::setUserAgentPart):
(WebCore::Internals::userAgentPartId): Deleted.
(WebCore::Internals::setUserAgentPartId): Deleted.
* Source/WebCore/testing/Internals.h:
* Source/WebCore/testing/Internals.idl:

Canonical link: <a href="https://commits.webkit.org/272743@main">https://commits.webkit.org/272743@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/320b38761c3f21b0c7a7947f22ef759b64114292

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/32872 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/11630 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/34748 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/35495 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/29697 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/33809 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/13970 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/8805 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/29150 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/33327 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/9803 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/29360 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/8542 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/8683 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/29316 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/36828 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/29860 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/29723 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/34823 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/8809 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/6767 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/32677 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/10499 "Built successfully") | [❌ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/28973 "Exiting early after 10 failures. 1093 tests run. 1 flakes") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/7634 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/9415 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/9431 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on Source/WebCore/dom/Element.cpp:
```diff
@@ -3894,14 +3894,14 @@ String Element::title() const
 
-const AtomString& Element::pseudo() const
+const AtomString& Element::userAgentPart() const
 {
-    return attributeWithoutSynchronization(pseudoAttr);
+    return attributeWithoutSynchronization(useragentpartAttr);
 }
```
Probably a follow-up, but this should return nothing outside of UA shadow trees (or better, assert that it's not called outside)

nt1m commented on Source/WebCore/dom/Element.cpp:
```diff
@@ -3894,14 +3894,14 @@ String Element::title() const
 
-void Element::setPseudo(const AtomString& value)
+void Element::setUserAgentPart(const AtomString& value)
 {
-    setAttributeWithoutSynchronization(pseudoAttr, value);
+    setAttributeWithoutSynchronization(useragentpartAttr, value);
 }
```
Probably a follow-up: this should assert that it's inside an UA shadow root.

nt1m reviewed (approved):
Looks great, thanks!

annevk committed:
Rename UserAgentPartIds to UserAgentParts
https://bugs.webkit.org/show_bug.cgi?id=267177

Reviewed by Tim Nguyen.

This also renames:

- pseudoAttr to useragentpartAttr
- Element::pseudo() to Element::userAgentPart()
- Element::setPseudo() to Element::setUserAgentPart()
- Internals::userAgentPartId() to Internals::userAgentPart()
- Internals::setUserAgentPartId() to Internals::setUserAgentPart()

And adjusts various variables and static functions accordingly.

* LayoutTests/editing/style/apply-style-atomic-expected.txt:
* LayoutTests/editing/style/apply-style-atomic-live-range-expected.txt:
* LayoutTests/fast/dom/HTMLMeterElement/meter-clone-expected.txt:
* LayoutTests/fast/dom/HTMLMeterElement/meter-clone.html:
* LayoutTests/fast/dom/HTMLMeterElement/meter-element-markup-expected.txt:
* LayoutTests/fast/dom/HTMLProgressElement/progress-clone-expected.txt:
* LayoutTests/fast/dom/HTMLProgressElement/progress-clone.html:
* LayoutTests/fast/dom/HTMLProgressElement/progress-element-markup-expected.txt:
* LayoutTests/fast/dom/nodesFromRect/resources/nodesFromRect.js:
(checkShadowContent):
* LayoutTests/fast/forms/caps-lock-indicator-width.html:
* LayoutTests/fast/forms/resources/common.js:
(getElementByPseudoId):
* LayoutTests/fast/forms/search-field-buttons-do-not-have-focus-rings-expected.html:
* LayoutTests/fast/forms/search-field-buttons-do-not-have-focus-rings.html:
* LayoutTests/fast/forms/validation-message-clone-expected.txt:
* LayoutTests/fast/forms/validation-message-clone.html:
* LayoutTests/fast/html/clone-range-expected.txt:
* LayoutTests/fast/html/clone-range.html:
* LayoutTests/fast/html/details-clone.html:
* LayoutTests/http/tests/media/hls/hls-webvtt-style-expected.txt:
* LayoutTests/http/tests/media/hls/hls-webvtt-style.html:
* LayoutTests/http/tests/media/track/track-webvtt-vertical-multi-line-expected.txt:
* LayoutTests/http/tests/media/track/track-webvtt-vertical-multi-line.html:
* LayoutTests/inspector/css/pseudo-element-matches.html:
* LayoutTests/media/modern-media-controls/media-controller/media-controller-single-container-expected.txt:
* LayoutTests/media/modern-media-controls/media-controller/media-controller-single-container.html:
* LayoutTests/media/modern-media-controls/media-controls/media-controls-display-above-captions-expected.txt:
* LayoutTests/media/modern-media-controls/media-controls/media-controls-display-above-captions.html:
* LayoutTests/media/modern-media-controls/text-tracks/text-tracks-height-no-controls.html:
* LayoutTests/media/modern-media-controls/text-tracks/text-tracks-height-with-controls.html:
* LayoutTests/media/track/track-cue-vertical-style.html:
* LayoutTests/media/track/track-in-band-layout.html:
* LayoutTests/media/track/track-user-stylesheet-override.html:
* LayoutTests/media/track/webvtt-region-position-expected.txt:
* LayoutTests/media/track/webvtt-region-position.html:
* LayoutTests/platform/ios/ios/plugin/youtube-flash-plugin-iframe-expected.txt:
* LayoutTests/platform/ios/ios/plugin/youtube-flash-plugin-iframe.html:
* LayoutTests/plugins/quicktime-plugin-replacement-expected.txt:
* LayoutTests/plugins/quicktime-plugin-replacement.html:
* LayoutTests/resources/dump-as-markup.js:
(Markup._get):
* LayoutTests/userscripts/insert-stylesheets-with-pseudo.html:
* Source/WebCore/CMakeLists.txt:
* Source/WebCore/DerivedSources-output.xcfilelist:
* Source/WebCore/DerivedSources.make:
* Source/WebCore/Headers.cmake:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/css/CSSPseudoSelectors.json:
* Source/WebCore/css/SelectorChecker.cpp:
(WebCore::SelectorChecker::matchRecursively const):
* Source/WebCore/css/process-css-pseudo-selectors.py:
* Source/WebCore/dom/Element.cpp:
(WebCore::Element::attributeChanged):
(WebCore::Element::userAgentPart const):
(WebCore::Element::setUserAgentPart):
(WebCore::Element::pseudo const): Deleted.
(WebCore::Element::setPseudo): Deleted.
* Source/WebCore/dom/Element.h:
* Source/WebCore/dom/mac/ImageControlsMac.cpp:
(WebCore::ImageControlsMac::createImageControls):
* Source/WebCore/html/BaseDateAndTimeInputType.cpp:
(WebCore::BaseDateAndTimeInputType::createShadowSubtree):
* Source/WebCore/html/ColorInputType.cpp:
(WebCore::ColorInputType::createShadowSubtree):
* Source/WebCore/html/FileInputType.cpp:
(WebCore::FileInputType::createShadowSubtree):
* Source/WebCore/html/HTMLAttributeNames.in:
* Source/WebCore/html/HTMLMeterElement.cpp:
(WebCore::setValueClass):
(WebCore::HTMLMeterElement::didAddUserAgentShadowRoot):
* Source/WebCore/html/RangeInputType.cpp:
(WebCore::RangeInputType::createShadowSubtree):
* Source/WebCore/html/SearchInputType.cpp:
(WebCore::updateResultButtonPseudoType):
* Source/WebCore/html/TextFieldInputType.cpp:
(WebCore::TextFieldInputType::createShadowSubtree):
(WebCore::TextFieldInputType::createDataListDropdownIndicator):
(WebCore::autoFillButtonTypeToAutoFillButtonPseudoClassName):
(WebCore::isAutoFillButtonTypeChanged):
(WebCore::TextFieldInputType::createContainer):
(WebCore::TextFieldInputType::createAutoFillButton):
(WebCore::TextFieldInputType::updateAutoFillButton):
* Source/WebCore/html/ValidationMessage.cpp:
(WebCore::ValidationMessage::buildBubbleTree):
* Source/WebCore/html/shadow/DateTimeEditElement.cpp:
(WebCore::DateTimeEditBuilder::visitLiteral):
(WebCore::DateTimeEditElement::create):
(WebCore::DateTimeEditElement::layout):
* Source/WebCore/html/shadow/DateTimeFieldElements.cpp:
(WebCore::DateTimeDayFieldElement::create):
(WebCore::DateTimeHourFieldElement::create):
(WebCore::DateTimeMeridiemFieldElement::create):
(WebCore::DateTimeMillisecondFieldElement::create):
(WebCore::DateTimeMinuteFieldElement::create):
(WebCore::DateTimeMonthFieldElement::create):
(WebCore::DateTimeSecondFieldElement::create):
(WebCore::DateTimeSymbolicMonthFieldElement::create):
(WebCore::DateTimeYearFieldElement::create):
* Source/WebCore/html/shadow/DetailsMarkerControl.cpp:
(WebCore::DetailsMarkerControl::create):
* Source/WebCore/html/shadow/MediaControlTextTrackContainerElement.cpp:
(WebCore::MediaControlTextTrackContainerElement::create):
* Source/WebCore/html/shadow/ProgressShadowElement.cpp:
(WebCore::ProgressInnerElement::create):
(WebCore::ProgressBarElement::create):
(WebCore::ProgressValueElement::create):
* Source/WebCore/html/shadow/SliderThumbElement.cpp:
(WebCore::SliderThumbElement::create):
(WebCore::SliderContainerElement::create):
* Source/WebCore/html/shadow/SpinButtonElement.cpp:
(WebCore::SpinButtonElement::create):
* Source/WebCore/html/shadow/SwitchThumbElement.cpp:
(WebCore::SwitchThumbElement::create):
* Source/WebCore/html/shadow/SwitchTrackElement.cpp:
(WebCore::SwitchTrackElement::create):
* Source/WebCore/html/shadow/TextControlInnerElements.cpp:
(WebCore::TextControlPlaceholderElement::create):
(WebCore::SearchFieldCancelButtonElement::create):
* Source/WebCore/html/track/TextTrackCue.cpp:
(WebCore::TextTrackCueBox::initialize):
(WebCore::tagPseudoObjects):
(WebCore::removeUserAgentPartAttributes):
(WebCore::TextTrackCue::getCueAsHTML):
(WebCore::TextTrackCue::rebuildDisplayTree):
(WebCore::removePseudoAttributes): Deleted.
* Source/WebCore/html/track/VTTCue.cpp:
(WebCore::VTTCue::obtainCSSBoxes):
* Source/WebCore/html/track/VTTRegion.cpp:
(WebCore::VTTRegion::getDisplayTree):
(WebCore::VTTRegion::prepareRegionDisplayTree):
* Source/WebCore/inspector/agents/InspectorDOMAgent.cpp:
(WebCore::InspectorDOMAgent::highlightSelector):
* Source/WebCore/page/CaptionUserPreferencesMediaAF.cpp:
(WebCore::CaptionUserPreferencesMediaAF::captionsStyleSheetOverride const):
* Source/WebCore/rendering/RenderTheme.cpp:
(WebCore::RenderTheme::autoAppearanceForElement const):
* Source/WebCore/style/ElementRuleCollector.cpp:
(WebCore::Style::ElementRuleCollector::matchPartPseudoElementRules):
(WebCore::Style::ElementRuleCollector::collectMatchingUserAgentPartRules):
* Source/WebCore/style/RuleData.cpp:
(WebCore::Style::determinePropertyAllowlist):
* Source/WebCore/style/StyleAdjuster.cpp:
(WebCore::Style::Adjuster::adjust const):
* Source/WebCore/style/StyleInvalidator.cpp:
(WebCore::Style::Invalidator::invalidateUserAgentParts):
* Source/WebCore/style/StyleResolver.cpp:
(WebCore::Style::elementTypeHasAppearanceFromUAStyle):
* Source/WebCore/style/StyleSharingResolver.cpp:
(WebCore::Style::SharingResolver::canShareStyleWithElement const):
* Source/WebCore/testing/Internals.cpp:
(WebCore::Internals::userAgentPart):
(WebCore::Internals::setUserAgentPart):
(WebCore::Internals::userAgentPartId): Deleted.
(WebCore::Internals::setUserAgentPartId): Deleted.
* Source/WebCore/testing/Internals.h:
* Source/WebCore/testing/Internals.idl:

Canonical link: https://commits.webkit.org/272743@main
